enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -197,8 +208,15 @@ class Meta:
         default=None,
         help_text="The datetime when this Subsidy is considered expired.  If null, this Subsidy is considered active."
         )
+    is_hidden = models.BooleanField(
+        default=False,
+        help_text="If set, this subsidy will be considered as soft-deleted or deactivated.",
+    )
```

Comment:
Since we've included this field in the filter for the default model manager, we'll want to add an index on it to keep all of the queries performant.